include/volk/volk_common.h

```diff
@@ -87,7 +87,7 @@
 ////////////////////////////////////////////////////////////////////////
 // The bit128 union used by some
 ////////////////////////////////////////////////////////////////////////
-#include <inttypes.h>
+#include <stdint.h>
```

Comment:
Without it, it does not work in macOS. When commenting out https://github.com/carlesfernandez/volk/blob/67f2f6600f7c19867c9e8b48bf31ea662c225b1f/lib/CMakeLists.txt#L156 , without that change, the error is:


```
...
-- x86* CPU detected
In file included from /Users/carlesfernandez/Documents/volk/build/lib/test_xgetbv.c:1:
In file included from /Users/carlesfernandez/Documents/volk/include/volk/volk_common.h:90:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/11.0.0/include/inttypes.h:30:15: fatal error: 'inttypes.h' file not found
#include_next <inttypes.h>
              ^~~~~~~~~~~~
1 error generated.
-- Compiler or linker missing xgetbv instruction, Overruled arch avx
...
```

The difference bewteen these includes is explained here https://stackoverflow.com/questions/7597025/difference-between-stdint-h-and-inttypes-h

Reply:
Since we don't need `inttypes.h` I guess, we can go for `stdint.h`. I wonder if this changed recently? I can only imagine someone used `inttypes` because of debugging.
I don't see the difference in your CI though.
Also, we should probably also set up Github Actions as an additional CI.

Reply:
The difference in the CI:

Without the change (branch `actions`), in configure step of `volk_macos` job:

```
...
-- x86* CPU detected
-- Compiler or linker missing xgetbv instruction, Overruled arch avx
-- Performing Test neon_compile_result
-- Performing Test neon_compile_result - Failed
...
```

With the change (branch`fix-avx-neon-detection-ci`), in configure step of `volk-macos` job:
```
...
-- x86* CPU detected
-- Performing Test neon_compile_result
-- Performing Test neon_compile_result - Failed
...
```

The same happens in the `volk-ubuntu` job (not related to this change in the include, though)